thesis-de/content-de.tex

```diff
@@ -1094,56 +1094,221 @@ \subsubsection{Durchführung Evaluierung Frontend}
 Im Nachfolgenden Punkt sind die einzelnen Softwarelösungen mit den einzelnen
 Anforderungen und Evaluierungsergebnissen aufgelistet:
 
-
+\paragraph{Grafana}
+\label{paragraph:grafana}
+Grafana ist eine von Torkel Ödegaard & Raintank Inc Entwickelte OpenSource
+Visualisierungssoftware. Sie bietet neben einfachen Visualisierungs Graphen noch
+weitere Ereignisfunktionen. Es arbeitet mit jedem TimeSeries
+Datenhaltungssystem, wie z.B. influxDB zusammen und ist Multi Tenant fähig.
+Dies bedeutet, dass sowohl ein LDAP als auch eine Benutzernamen plus Kennwort
+Lösung eingesetzt werden kann. Somit können z.B. Personen aus der Führungsebene
+Eingeschränkte Zugriffe aus das System erhalten, um hier Analysen oder
+Marketing Statistiken durchführen zu können. Zusätzliche zu der Abfrage mit
```

Comment:
Analysen werden in der Datenhaltung durchgeführt. Bei dir werden Ergebnisse interpretiert.